code/game/objects/items/weapons/storage/bags.dm

```diff
@@ -460,6 +460,28 @@
 	return ..()
 
 
+/obj/item/storage/bag/tray/cookies_tray
+	var/cookie = /obj/item/reagent_containers/food/snacks/cookie
+
+/obj/item/storage/bag/tray/cookies_tray/New() /// By Azule Utama, thank you a lot!
+	..()
+	for(var/i in 1 to 6)
+		var/obj/item/C = new cookie(src)
+		handle_item_insertion(C)    // Done this way so the tray actually has the cookies visible when spawned
+	rebuild_overlays()
+/*

```

Comment:
No leftover code please.

Reply:
Oops, I had some commented code while testing ad I forgot to delete it later on. Gonna fix that